metrics_v2/handler/alerts.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import datetime
+import logging
+import pytz
+import typing
+
+import jinja2
+from sendgrid.helpers import mail
+
+# Make sure these match the names of your Cloud Secrets. Refer to the README
```

Comment:
Not sure what the word "these" is referring to

Reply:
also I think this README doesn't exist yet

Reply:
I will copy over the instructions here: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/tree/master/metrics_handler#optional-set-up-notification-emails